app/src/main/java/com/android/doctorapp/ui/profile/ProfileFragment.kt

```diff
@@ -63,6 +84,31 @@ class ProfileFragment : BaseFragment<FragmentProfileBinding>(R.layout.fragment_p
 //                showImagePicker(startForProfileImageResult)
 //            })
 //        }
+
+        viewModel.getUserProfileData()
+        viewModel.isEdit.observe(viewLifecycleOwner) {
+            if (it) {
+                findNavController().navigate(
+                    R.id.action_user_profile_to_updateUserProfile
+                )
+            }
+        }
+        viewModel.phoneClick.observe(viewLifecycleOwner) {
```

Comment:
make common method for phone number. phone number as parameter of function. That is help full to other screen if needed.